### PR TITLE
Search button nav

### PR DESCRIPTION
**Pull request by lindakw**
Added navigation to search button on Main/AQI page. 
-clicking the magnifying glass will take users back to the search page

Also added navigation to the main search button on the search page.
-after clicking search user will be taken to the Main/AQI page

### CODE REVIEW

**Comment by kmausolf** [comment]
Judging by the large diff here in the `package-lock`, we need to ensure everyone on this team is using the same version of node.  @lindakw , can you determine what node version your team wants to use?  After you've done that, switch to that version in `nvm`, then run `npm install` again.  That's what's causing that large diff, the package lock is trying to be compatible with too many node versions.

File: api/airQualityAPI.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  }),
+  endpoints: (build) => ({
+    getAQI: build.query({
+      query: (city) => `${city}/?token=${API_KEY}`
+    })
+  })
+})
+
+
+export const {useGetAQIQuery} = airQualityAPI
```
Comment (kmausolf) [inline api/airQualityAPI.ts]: Well done on the implementation!  Do we need those other API urls?  I would suggest putting them in the `.private.js` file and just importing them where you need them.

File: app/store.ts [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
-})
\ No newline at end of file
+  reducer: {
+     [airQualityAPI.reducerPath]: airQualityAPI.reducer,
+  },
+  middleware: (getDefaultMiddleware) => 
+    getDefaultMiddleware().concat(airQualityAPI.middleware),
+})
+
+setupListeners(store.dispatch)
```
Comment (kmausolf) [inline app/store.ts]: Store setup looks perfect!

File: screens/search_screen/search.tsx [hunk 3]
```diff
@@ -24,23 +26,47 @@ const Search: FC<SearchScreenProps> = (props) => {
     (async () => {
       let { status } = await Location.requestForegroundPermissionsAsync();
       if (status !== "granted") {
-        alert("Permission to access location was denied");
+        alert("Permission to access location was denied"); useEffect(() => {
+          (async () => {
+            let { status } = await Location.requestForegroundPermissionsAsync();
+            if (status !== "granted") {
+              alert("Permission to access location was denied");
```
Comment (kmausolf) [inline screens/search_screen/search.tsx:33]: I think we want to use the `Alert` component here instead.  Is there a reason we have to use the function?

**Review by kmausolf — COMMENTED** [review]
Once we're sure that everyone is on the same node version, I think this is more or less good to go.  I've included some suggestions as well.

**Review by Amele-A — APPROVED** [review]
looks good!

**Review by kmausolf — CHANGES_REQUESTED** [review]
We need to change the base branch to development before merge.  Once this is done, I'll remove the hold.  Just want to ensure code isn't getting merged to the wrong place.

**Review by kmausolf — COMMENTED** [review]
There we go!

**Review by kmausolf — APPROVED** [review]
Oops, sorry, forgot to approve

**Pull request merged**